response_operations_ui/controllers/collection_instrument_controllers.py

```diff
@@ -182,6 +182,24 @@ def unlink_collection_instrument(ce_id, ci_id):
     return True
 
 
+def delete_seft_collection_instrument(ci_id: str) -> bool:
+    """Deletes a SEFT collection instrument
```

Comment:
I have reduced logging compared to others, I just don't think we need the noise, we just need to know the cause if it fails. There will also be a request call in the logs anyway